src/js/components/Tabs/README.md

```diff
@@ -176,6 +176,36 @@ div
 ```
 ## Theme
   
+**global.borderSize**
+
+The size of the border Expects `object`.
+
+Defaults to
+
+```
+undefined
+```
+
+**global.colors.border**
+
+color of the border Expects `string`.
```

Comment:
Let's fix the description here to be `color of the Tabs border`, so no one will confuse it with the `Tab` border